.gitlab/build_ruby.yml

```diff
@@ -7,7 +7,7 @@
 # This is the shared configuration of jobs for ruby
 .on_ruby:
   variables:
-    SCHEDULER_PARAMETERS: "--res=ci --exclusive=user --deadline=now+1hour -N1 -t ${ALLOC_TIME}"
+    SCHEDULER_PARAMETERS: "--reservation=ci --exclusive=user --deadline=now+1hour -N1 -t ${ALLOC_TIME}"
```

Comment:
This change is blocking other PRs. Can it be pulled into its own PR? Or can this be PR be merged?

Reply:
I'll merge it.